base.js

```diff
@@ -79,149 +94,164 @@ const baseRules = {
 const { js: jsExtensions, ts: tsExtensions } = require('./extensions');
 const allExtensions = [...jsExtensions, ...tsExtensions];
 
-/** @type {import('eslint').Linter.Config} */
-const baseConfig = {
-  parser: '@babel/eslint-parser',
-  parserOptions: {
-    requireConfigFile: false,
-    sourceType: 'module',
-  },
-  root: true,
-  env: {
-    node: true,
+const settings = {
+  'import-x/resolver': {
+    typescript: true,
+    node: {
+      moduleDirectory: [root, 'node_modules'],
```

Comment:
I wonder if we need to put `root` here anymore? It was added [a long time ago](https://github.com/seek-oss/eslint-config-seek/pull/5), and may not be an issue anymore.

A lot of references to `import-x/resolver` [on github](https://github.com/search?q=language%3AJavaScript+%22import-x%2Fresolver%22&type=code&p=1) simply specific `typescript: true, node: true`.